code/modules/mining/fulton.dm

```diff
@@ -168,6 +168,10 @@ GLOBAL_LIST_EMPTY(total_extraction_beacons)
 	icon_state = "folded_extraction"
 
 /obj/item/fulton_core/attack_self(mob/user)
+	if(istype(get_area(user), /area/misc/hilbertshotel))
+		to_chat(user, span_userdanger("Something about this place is interfering with the beacon. It is unsafe to deploy."))
+		return .
```

Comment:
```suggestion
		return
```
`return` would always return the value of `.` anyways if something else isn't passed to it
also `.` isn't set here regardless